clio/example-third-party-application
# setup/config changes

Made 2 setup/config changes:
1. Added the missing `Gemfile.lock`
1. Started storing our environment variables in `.env` instead of `config/local_env.yml`

**How to test?**
_Use a fresh copy of the repo and don't use an IDE for this test! RubyMine was generating the Gemfile.lock automatically, so the setup was working without it._

In a terminal, do the following 
```
git clone git@github.com:clio/example-third-party-application.git
git checkout dc-setup 
```

Then, in a text editor, update your `.env` file according to the readme. 

Back in the terminal
```
docker compose build
docker compose run sb_app rake db:create
docker compose up
```

Go to localhost:3013 and you should be able to successfully log into Switchboard.
